src/runtime_src/core/include/xclbin.h

```diff
@@ -329,6 +329,18 @@ extern "C" {
         FAST_ADAPTER = 5
     };
 
+    // m_functional
+    enum FUNCTIONAL {
+        FC_DPU = 0,
+        FC_PREPOST = 1
```

Comment:
I have see folks add a comma even at the last element.  Doing so allows for diffs be clean when adding additional enumeration values in future work.
```suggestion
        FC_PREPOST = 1,
```

Reply:
change the code as suggested